test/integration/admin_dashboard_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class AdminDashboardTest < ActionDispatch::IntegrationTest
+  test 'logged in admin sees dashboard page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.